sdf-schema.json

```diff
@@ -251,6 +254,10 @@
             "widthInBits": {
               "type": "number"
             },
+            "subtype":{
```

Comment:
This is where HJSON would be helpful; having difficulty seeing "forest" for "trees" here. What is "subtype" modifying?

Reply:
subtype is a quality applicable to ODMdata that is used to constrain a data element to some predefined type, like uint8

We will eventually want to define the enum of allowable types, but initially populated with a couple of subtypes that are used by OMA

We may also need to make it available to individual elements of a complex data item